resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -92,6 +93,7 @@ export default {
       filter: "",
       markCategory: false,
       fromProcessList: false,
+      categoryCount: 0,
```

Comment:
Please use the permission in order to see the new option, this is not necesary

Reply:
This is necessary to initialize categoryCount variable